Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -16,6 +16,10 @@ class EndOfYearDataManager {
                                             lastPlaybackInteractionDate IS NOT NULL AND lastPlaybackInteractionDate BETWEEN strftime('%s', '\(startDate)') and strftime('%s', '\(endDate)')
                                            """
 
+    private lazy var listenedEpisodesPreviousYear = """
+                                            lastPlaybackInteractionDate IS NOT NULL AND lastPlaybackInteractionDate BETWEEN strftime('%s', '2021-01-01') and strftime('%s', '"2023-01-01"')
```

Comment:
My stats were showing that I didn't listen to anything. 

```suggestion
                                            lastPlaybackInteractionDate IS NOT NULL AND lastPlaybackInteractionDate BETWEEN strftime('%s', '2021-01-01') and strftime('%s', '2023-01-01')
```

Reply:
I fixed that on an upcoming PR but did it here: b8bc790f66077a82855fc126ae594b6601af8768